File: program6.c
/*
Algorithm
    Start   
        Accept 1st number as No1
        Accept 2nd number as No2
        Perform Addition of No1 and No2
        Display the result
    Stop
*/

#include<stdio.h>

int main()
{
    // Variable creation with default values
    float No1 = 0.0f;
    float No2 = 0.0f , Result = 0.0f;

    printf("Enter First number : \n");
    scanf(" %f ", &No1);

    printf("Enter Second number  : \n");
    scanf("%f" , &No2);

    Result = No1 + No2 ;

    printf("Addition is : %f \n" , Result);

    return 0;
}

File: program6.java
import java.util.Scanner;

class program6 
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        //intialize variables with default values
        float i = 0.0f;
        float j = 0.0f;
        float k = 0.0f;

        System.out.println("Enter First Number : ");
        i = sobj.nextFloat();

        System.out.println("Enter Second Number : ");
        j = sobj.nextFloat();

        k = i + j;

        System.out.println("Addition is : "+k);

        sobj.close();
    }    
}
